src/libs/actions/Report.js

```diff
@@ -1106,12 +1107,16 @@ function addAction(reportID, text, file) {
     const textForNewComment = isAttachment ? '[Attachment]'
         : htmlForNewComment.replace(/((<br[^>]*>)+)/gi, ' ').replace(/<[^>]*>?/gm, '');
 
+    // Remove HTML from text when applying optimistic offline comment
+    const lastMessageText = isAttachment ? Localize.translateLocal('common.attachment') : htmlForNewComment.replace(/((<br[^>]*>)+)/gi, ' ').replace(/<[^>]*>?/gm, '');
+
```

Comment:
Duplicate code....

Reply:
Not exactly. textForNewComment is for ReportAction. '[Attachment]' is used as text in message in reportAction from server as well. So keeping it as it is. What we can do here is store `htmlForNewComment.replace(/((<br[^>]*>)+)/gi, ' ').replace(/<[^>]*>?/gm, '');` to a local variable and use it in both place.

Reply:
I do not understand textForNewComment is purely use to store the data on Onyx...

Reply:
Yes. textForNewComment is used to store message.text of reportAction in Onyx. Later when Response from server is received after adding comment with API.Report_AddComment, it(response from server) also contains ''[Attachment]" as message.text.

Reply:
Ok, I see. Do we have any other way of checking that by looking at the server response?

Reply:
No. I don't think there is any.

Reply:
ok Then please dry this up.

Reply:
Sorry. I don't understand.

Reply:
Move this `htmlForNewComment.replace(/((<br[^>]*>)+)/gi, ' ').replace(/<[^>]*>?/gm, '')` into a variable and then Add a comment why `textForNewComment = isAttachment ? '[Attachment]'` needs to have `[Attachement]`.